packages/sanity/src/tasks/i18n/resources.ts

```diff
@@ -42,50 +40,81 @@ const tasksLocaleStrings = defineLocalesResources('tasks', {
+  'empty-state.list.no-active-document.heading': "Open a document to see it's task",
+  /** The text in the tasks sidebar, when viewing the document tab, but there is not an active document*/
+  'empty-state.list.no-active-document.text': 'Tasks on your active document will be shown here.',
+  /** The heading in the tasks sidebar, in the subscriber tab, when the user is not subscribed to any task*/
   'empty-state.list.subscribed.heading': "You haven't subscribed to any tasks",
+  /** The text in the tasks sidebar, in the subscriber tab, when the user is not subscribed to any task*/
   'empty-state.list.subscribed.text':
     'When you create, modify, or comment on a task you will be subscribed automatically',
 
+  /** The heading in the tasks sidebar, in the assigned tab, under the closed details, when it's empty.*/
   'empty-state.status.list.closed.assigned.heading': 'No completed tasks',
+  /** The text in the tasks sidebar, in the assigned tab, under the closed details, when it's empty.*/
   'empty-state.status.list.closed.assigned.text': 'Your tasks marked done will show up here',
+  /** The heading in the tasks sidebar, in the document tab, under the closed details, when it's empty.*/
   'empty-state.status.list.closed.document.heading': 'No completed tasks',
+  /** The heading in the tasks sidebar, in the subscribed tab, under the closed details, when it's empty.*/
   'empty-state.status.list.closed.subscribed.heading': 'No completed tasks',
+  /** The text in the tasks sidebar, in the subscribed tab, under the closed details, when it's empty.*/
   'empty-state.status.list.closed.subscribed.text':
     'Tasks you subscribe to marked done will show up here',
 
+  /** The heading in the tasks sidebar, in the assigned tab, under the open details, when it's empty.*/
   'empty-state.status.list.open.assigned.heading': "You're all caught up",
+  /** The text in the tasks sidebar, in the assigned tab, under the open details, when it's empty.*/
   'empty-state.status.list.open.assigned.text': 'New tasks assigned to you will show up here',
+  /** The heading in the tasks sidebar, in the document tab, under the open details, when it's empty.*/
   'empty-state.status.list.open.document.heading': 'No tasks on this document',
+  /** The heading in the tasks sidebar, in the subscribed tab, under the open details, when it's empty.*/
   'empty-state.status.list.open.subscribed.heading': 'No subscribed tasks',
+  /** The text in the tasks sidebar, in the subscribed tab, under the open details, when it's empty.*/
   'empty-state.status.list.open.subscribed.text': 'Tasks you subscribe to will show up here',
 
   /** Text used in the assignee input when there is no user assigned */
-  'form.input.assignee.no-user-assigned.text': 'Not assigned',
+  'form.input.assignee.no-user-assigned.text': 'Unassigned',
+  /** Text used in the assignee input tooltip when there is no user assigned */
+  'form.input.assignee.no-user-assigned.tooltip': ' Set assignee',
```

Comment:
Is the leading space here intentional? If so we should move it to the source code instead of the translation strings, since it is not obvious to a translator wether or not they need to include one. If it is an oversight, the space should be removed (this also goes for `form.input.assignee.user-assigned.tooltip`)

Reply:
Oversight, will update. Thanks